src/Language/pl/Auth.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter Shield.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+return [
+    // Exceptions
+    'unknownAuthenticator'  => '{0} nie jest poprawnym autentykatorem.',
+    'unknownUserProvider'   => 'Nie można określić User Provider do użycia.',
+    'invalidUser'           => 'Nie można zlokalizować określonego użytkownika.',
+    'bannedUser'            => 'Nie możesz cię zalogować, ponieważ jesteś obecnie zbanowany.',
+    'logOutBannedUser'      => 'Zostałeś wylogowany, ponieważ zostałeś zbanowany.',
+    'badAttempt'            => 'Nie można Cię zalogować. Sprawdź swoje dane uwierzytelniające.',
+    'noPassword'            => 'Nie może zweryfikować użytkownika bez hasła.',
+    'invalidPassword'       => 'Nie można cię zalogować. Sprawdź swoje hasło.',
+    'noToken'               => 'Każde żądanie musi mieć token w nagłówku {0}.',
+    'badToken'              => 'Token dostępu jest nieprawidłowy.',
+    'oldToken'              => 'Token dostępu wygasł.',
+    'noUserEntity'          => 'User Entity jest niezbędne do sprawdzania poprawności hasła.',
+    'invalidEmail'          => 'Nie można zweryfikować zgodności adresu e-mail z zarejestrowanym adresem e-mail.',
+    'unableSendEmailToUser' => 'Przepraszamy, był problem z wysłaniem wiadomości e-mail. Nie mogliśmy wysłać wiadomości e-mail do "{0}".',
+    'throttled'             => 'Zbyt wiele żądań z tego adresu IP. Możesz spróbować ponownie za {0} sekund.',
+    'notEnoughPrivilege'    => 'Nie masz uprawnień niezbędnych do wykonania żądanej operacji.',
+    // JWT Exceptions
+    'invalidJWT'     => 'Token jest nieważny.',
+    'expiredJWT'     => 'Token wygasł.',
+    'beforeValidJWT' => 'Token nie jest jeszcze dostępny.',
+
+    'email'           => 'Adres e-mail',
+    'username'        => 'Nazwa użytkownika',
+    'password'        => 'Hasło',
+    'passwordConfirm' => 'Hasło (ponownie)',
+    'haveAccount'     => 'Posiadasz już konto?',
+    'token'           => 'Token',
```

Comment:
If the translation is correct, add `$excludedLocaleKeyTranslations` in the test case class.
See https://github.com/codeigniter4/shield/pull/913#issuecomment-1766414597